packages/terra-embedded-content-consumer/src/EmbeddedContentConsumer.jsx

```diff
@@ -96,18 +107,36 @@ class EmbeddedContentConsumer extends React.Component {
       onAuthorize,
       options,
       eventHandlers,
+      intl,
       ...customProps
     } = this.props;
 
     return (
-      <div
-        {...customProps}
-        ref={(element) => { this.embeddedContentWrapper = element; }}
-      />
+      <div className={cx('container')}>
+        <VisuallyHiddenText
+          className={cx('visually-hidden-text')}
+          text={intl.formatMessage(
+            { id: 'Terra.embeddedContentConsumer.beginEmbeddedContent' },
+            { name: options?.iframeAttrs?.title },
```

Comment:
Won't this read the title twice, since we're pushing towards encouraging consumers to always add the iframe title in: #1776 ?

CC: @eawww

Reply:
Yes, both JAWS and VO will announce the title on the iframe if one is provided. I am okay with removing it from the messaging here if we think including it will make things too verbose.

CC: @scottwilmarth @eawww

Reply:
@sdadn I discussed this in scrum with @scottwilmarth and he said that we should keep the title here as that is what will give users context on what the embedded content is about if it fails to load.